server/database.sql

```diff
@@ -26,7 +28,12 @@ CREATE TABLE users (
       VALUES ('goodness', 'aig', 'goodness@gmail.com','password', 'male', 'None', 'develope', true ,'No 7 eyaen, benin city');
 
       CREATE TABLE gifs (
-      id SERIAL PRIMARY KEY,
-      title VARCHAR(128) NOT NULL,
-      imageUrl VARCHAR(128) NOT NULL,
-      createdOn DATE NOT NULL );
\ No newline at end of file
+        gif_id SERIAL PRIMARY KEY,
+        title VARCHAR(128) NOT NULL,
+        image_url VARCHAR(128) NOT NULL,
+        public_id varchar NOT NULL,
+        created_at timestamp with time zone NOT NULL,
+        created_by VARCHAR(128) NOT NULL,
```

Comment:
please remove the created by, you are not sending the that and it will cause an error